levels/lovelace_tower/events.js

```diff
@@ -7,347 +7,78 @@ const INITIAL_STATE = {
   obj3Complete: false,
   obj4Complete: false,
   obj5Complete: false,
```

Comment:
Where are the `obj#Complete` being set? Also, would it make sense to use the `world.isObjectiveCompleted` method for performing these checks?

Reply:
Absolutely to the world.isObjectiveCompleted